frontend/components/Util/UtilWrite.js

```diff
@@ -0,0 +1,81 @@
+import AWS from 'aws-sdk';
+import { Amplify, Auth } from 'aws-amplify';
+import { TestConstants } from '../../../amplify/ts-code/__dev__/db/DBTestConstants';
+
+import awsconfig from '../../../src/aws-exports';
+
+const ENV_SUFFIX = '-alpha'
+const ENV_REGION = 'us-west-2'
+
+function UtilWrite(props) {
+    Amplify.configure(awsconfig);
+
+    Auth.signIn({
+        username: TestConstants.EMAIL,
+        password: TestConstants.PASSWORD
+    }).then(() => Auth.currentCredentials())
+    .then((credentials) => credentials.authenticated);
+
+
+    Auth.currentCredentials()
+    .then((credentials) => {
+        AWS.config.credentials = credentials
+        const lambda = new AWS.Lambda({
+            credentials: credentials,
+            region: ENV_REGION
+        })
+        AddNewItem(lambda);
+    });                
+
+    var paramsAdd = {
```

Comment:
Do the docs specify that you must use `var`? If so, we can keep this, but if not let's use `const` instead.